app/components/users/profile/attributes_component.html.erb

```diff
@@ -0,0 +1,37 @@
+<%=
+  component_wrapper do
+    flex_layout(mt: 3) do |details_container|
+      details_container.with_row do
+        flex_layout(align_items: :center, justify_content: :space_between) do |heading|
+          heading.with_column(flex: 1) do
+            render(Primer::Beta::Heading.new(tag: :h4)) { t(:label_information) }
+          end
+
+          if User.current.admin?
+            heading.with_column do
+              render(Primer::Beta::Link.new(
+                href: edit_user_path(@user),
+                scheme: :secondary
+              )) { t(:label_manage) }
+            end
+          end
```

Comment:
If the current user is not the admin but the displayed user, we could like to the account page.

Here, a 'manage' link is displayed whereas on the meetings sidebar, a gear-icon is used. I think we should settle on one of these.